test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -957,7 +1029,7 @@ Describe "Invoke-RestMethod tests" -Tags "Feature" {
     ## 'HttpClientHandler.ServerCertificateCustomValidationCallback' currently doesn't work in netcoreapp2.0 on Mac at all.
     ## This is tracked by powershell issue #3648.
     It "Validate Invoke-RestMethod -SkipCertificateCheck" -Pending:$IsOSX {
-
+        
```

Comment:
Local and online diff sometimes surprise us.
Here may be a real difference in '`r`n' and ``n`.